### ISSUE TITLE
Sounds gets stuck

### ISSUE THREAD

**Issue by maryfries** [jmoenig/Snap--Build-Your-Own-Blocks#1712]
I had open a project with sound blocks. I went to open another project, and I somehow managed to click a script before the new project loaded. Now the sound script from the old project is stuck on a single note even though the new project is loaded. Opening new projects does not help. I had to close the tab to get the sound to stop.

**Comment by cycomachead** [comment]
So, if I understand this right:

If a sound is playing, and you open a new project, it keeps on playing, and the stop block doesn't affect it?

I'll test later, but this sounds like it should be possible to reproduce, and I think even makes sense with my own understanding of how things work.

**Comment by maryfries** [comment]
Yes. That's not quite what I did, but that seems to cause the bug too. In fact, I can even play a note, open a new project, play another note, open another new project, add yet another note, and so on building up a chord of notes all played out of separate projects that are not running. They never stop, and the stop button has no control.

I _think_ I actually recall seeing something else keep going after launching a new project, but I don't recall what it was. So, you might keep in mind the possibility that it's not just sound...

**Comment by cycomachead** [comment]
> I think I actually recall seeing something else keep going after launching a new project, but I don't recall what it was. So, you might keep in mind the possibility that it's not just sound...

Yeah, that's what I was thinking too.
